src/jspm-hmr-server.ts

```diff
@@ -66,7 +66,7 @@ export function createServer(options: ServerOptions): JspmHmrServer {
 
     const proxy = httpProxy.createProxyServer();
     app.use(proxyRoute, (req, res) => {
-      req.url = req.baseUrl;
+      req.url = `${req.baseUrl}${req.url}`;
```

Comment:
Looks good for cases that have requests only for base route e.g. `/search?q=something`
However I'm not sure it would work for deeper routes like e.g. `/api/products/search?q=something`, can you please confirm that?